anatomy/tracks/concept-exercises.md

```diff
@@ -72,6 +72,19 @@ exercises
         └── CarsAssemblyTests.cs (tests)
 </pre>
 
+### Minimal valid spec
+
+We favor an "optimistic merging" approach to new exercises, where tracks can develop exercises in a "work in progress" state.
+The minimal valid state, which will pass configlet and allow you to merge is:
```

Comment:
Is this supported by configlet already? :)

Reply:
No, I don't think so. At the moment configlet isn't checking for some of these, and/or warning for some of them rather than errorring. That's giving us problems such as tracks where they have `wip` exercises in the config.json but no corresponding exercise folders, which breaks the syncer.

The aim is for this PR to confirm the spec, then Erik/ee7 can get it into confliget :)